### PR TITLE
Bug 1325024, added deployer template parameters

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1325024

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh @tpoitras @bfallonf please peer review  :bow:

@TheDiemer PTAL

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -376,6 +376,37 @@ The deployer template parameter options and their defaults are listed in the
...
+|Parameter |Description
+
+|`*METRIC_DURATION*`
+|The number of days metrics should be stored.
+
+|`*CASSANDRA_PV_SIZE*`
+|The persistent volume size for each of the Cassandra nodes.
+
+|`*USE_PERSISTENT_STORAGE*`
+|Set to *true* for persistent storage, set to *false* to use non-persistent storage.
```
Comment (bfallonf) [inline install_config/cluster_metrics.adoc]: The comma above should be a semicolon or period, or divided with an "or": "...persistent storage, or set to..."

File: install_config/cluster_metrics.adoc [hunk 2]
```diff
@@ -376,6 +376,37 @@ The deployer template parameter options and their defaults are listed in the
...
+
+|`*MASTER_URL*`
+|Internal URL for the master, for authentication retrieval.
+
+|`*IMAGE_VERSION*`
+|Specify version for metrics components. (For example, for *openshift/origin-metrics-deployer:latest*, set version to *latest*.)
+
+|`*IMAGE_PREFIX*`
+|Specify prefix for metrics components. (For example, for *openshift/origin-metrics-deployer:latest*, set prefix to *openshift/origin-*.)
+|===
```
Comment (bfallonf) [inline install_config/cluster_metrics.adoc]: I don't think the above two would need parenthesis.

**Comment by bfallonf** [comment]
@ahardin-rh Two nits. Otherwise, LG!

File: install_config/cluster_metrics.adoc (hunk 1 above)
Comment (tpoitras) [inline install_config/cluster_metrics.adoc]: +1 :rainbow:

File: install_config/cluster_metrics.adoc [hunk 3]
```diff
@@ -376,6 +376,37 @@ The deployer template parameter options and their defaults are listed in the
...
+|The persistent volume size for each of the Cassandra nodes.
+
+|`*USE_PERSISTENT_STORAGE*`
+|Set to *true* for persistent storage, set to *false* to use non-persistent storage.
+
+|`*REDEPLOY*`
+|If set to *true*, the deployer will try to delete all the existing components before trying to redeploy.
+
+|`*HAWKULAR_METRICS_HOSTNAME*`
+|External host name where clients will reach Hawkular Metrics.
```
Comment (tpoitras) [inline install_config/cluster_metrics.adoc]: s/will/can

File: install_config/cluster_metrics.adoc (hunk 2 above)
Comment (tpoitras) [inline install_config/cluster_metrics.adoc]: +1 :rabbit:

**Comment by TheDiemer** [comment]
@ahardin-rh LGTM!

**Comment by tpoitras** [comment]
@ahardin-rh LGTM, too! :octocat:

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/cluster_metrics.adoc#install-config-cluster-metrics[Enabling Cluster Metrics]
|Added xref:../install_config/cluster_metrics.adoc#deployer-template-parameters[deployer template parameters].
%

**Pull request merged**